manim/__main__.py

```diff
@@ -73,7 +74,20 @@ def main():
     else:
         config.digest_args(args)
         input_file = config.get_dir("input_file")
-        if config["use_webgl_renderer"]:
+
+        if config["use_opengl_renderer"]:
+            from manim.renderer.opengl_renderer import OpenGLRenderer
+
+            for SceneClass in scene_classes_from_file(input_file):
+                try:
+                    renderer = OpenGLRenderer()
+                    scene = SceneClass(renderer)
+                    scene.render()
+                except Exception:
+                    print("\n\n")
+                    traceback.print_exc()
+                    print("\n\n")
+        elif config["use_webgl_renderer"]:
```

Comment:
IMO I would think the default should be called cairo.

Reply:
I agree with this too, but due to the large number of places where the flag is read it will end up being a pretty big PR, so I'd rather follow up with another one.

Reply:
Once this PR is merged, I'll implement this idea in #1013/handle any merge conflicts. *Hopefully* we won't have to refactor the flag in argparse as that'd be a duplicate effort.